atmosphere/operator/controllers/cloud.py

```diff
@@ -91,6 +91,15 @@ def create_fn(namespace: str, name: str, spec: dict, **_):
             ),
         )
 
+    flow.add(
+        tasks.ApplyNamespaceTask(
+            name=constants.NAMESPACE_OPENSTACK,
+            provides="openstack_namespace",
+        ),
+        tasks.ApplyHelmReleaseTask(
+            config=constants.HELM_RELEASE_INGRESS_NGINX,
+        ),
```

Comment:
this creates a regression, it is not possible to "disable" ingress-nginx with this...

Reply:
and make sure there's a way to do overrides too